sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -2956,6 +2956,21 @@ object GpuOverrides extends Logging {
               target: Option[Expression] = None): GpuExpression =
             GpuStringTrimRight(column, target)
         }),
+    expr[StringTranslate](
+      "StringTranslate operator",
```

Comment:
This should just be the definition of the function. You can just use the copy from here: https://spark.apache.org/docs/2.3.0/api/sql/index.html#translate

Reply:
Looks like we are pretty inconsistent in what we put into `desc`. Not for this PR, but I wonder if at least for *functions* we should just use permalinks to the Spark SQL doc?